src/test/java/org/padaiyal/utilities/aayvalar/statistics/StatisticsUtilityTest.java

```diff
@@ -71,16 +73,16 @@ private Number convertToNumber(String numericType, String number) {
   @ParameterizedTest
   @CsvSource({
       // Test different numeric types
-      "int,list,5.0,ARITHMETIC,'1,2,3,4,5,6,7,8,9'",
-      "long,list,5.0,ARITHMETIC,'1,2,3,4,5,6,7,8,9'",
-      "float,list,3.62,ARITHMETIC,'1.5, 2.3, 0.9, 8.3, 5.1'",
-      "double,list,3.62,ARITHMETIC,'1.5, 2.3, 0.9, 8.3, 5.1'",
+      "int,list,5.0,ARITHMETIC,'[1,2,3,4,5,6,7,8,9]'",
```

Comment:
Do the test pass with the square brackets?

Reply:
Yup, it does.